src/app/newSettings/Sections/Workspace/Teams/components/CreateTeamDialog.tsx

```diff
@@ -0,0 +1,51 @@
+import Modal from 'app/shared/components/Modal';
+import Input from 'app/shared/components/Input';
+import Button from 'app/shared/components/Button/Button';
+import { useTranslationContext } from 'app/i18n/provider/TranslationProvider';
+
+interface CreateTeamDialogProps {
+  isOpen: boolean;
+  onClose: () => void;
+  newTeamName: string;
+  setNewTeamName: (name: string) => void;
+  isLoading: boolean;
+  createTeam: () => void;
+}
+
+const CreateTeamDialog: React.FC<CreateTeamDialogProps> = ({
+  isOpen,
+  onClose,
+  newTeamName,
+  setNewTeamName,
+  isLoading,
+  createTeam,
+}) => {
+  const { translate } = useTranslationContext();
+  return (
+    <Modal isOpen={isOpen} onClose={onClose} width="w-96">
+      <h2 className="mb-5 text-2xl font-medium text-gray-100">
+        {translate('preferences.workspace.teams.createTeamDialog.title')}
+      </h2>
+      <Input
+        label={translate('preferences.workspace.teams.createTeamDialog.title')}
```

Comment:
Here is also the title of the modal, it has to show something like "Team name", right?

Reply:
If I understand you correctly, that is shown in the team detail, which is the next thing I'm working on, it will be included in the next PR.